nil/internal/execution/proposal.go

```diff
@@ -8,15 +8,15 @@ import (
 )
 
 type Proposal struct {
```

Comment:
We need json tags for a proposal.

https://github.com/NilFoundation/nil/blob/b3a123a44ede3cffe12b327f0d1ef762db9d6ca5/nil/internal/execution/block_generator.go#L180